lib/regions.ts

```diff
@@ -1,12 +1,14 @@
 export class Region {
   readonly trackUrl: string;
   readonly apiUrl: string;
+  readonly trackPushUrl: string;
 
-  constructor(trackUrl: string, apiUrl: string) {
+  constructor(trackUrl: string, apiUrl: string, trackPushUrl: string) {
```

Comment:
To maintain backwards compatibility, this should be optional. I doubt anyone is creating their own Regions, but just in case!